cmake/RunTests.cmake

```diff
@@ -20,12 +20,16 @@ if(NVIM_PRG)
   set(ENV{NVIM_PRG} "${NVIM_PRG}")
 endif()
 
-if(DEFINED ENV{TEST_FILE})
+if(TEST_PATH)
+  set(TEST_PATH "${TEST_PATH}")
+elseif(DEFINED ENV{TEST_FILE})
```

Comment:
> Adding another layer isn't a desirable way to do that.

@justinmk, until we agree on that, what do you think of re-write parts of `RunTest.cmake` to work with this then?

Reply:
if the rewrite leads to something useful then we want that :) If it leads to churn without a purpose then I want to save you time by cautioning against that.

Reply:
as this is still a very rough draft, and since you don't seem that convinced, I'll leave it open for a while.